.github/workflows/docker-build-push.yaml

```diff
@@ -0,0 +1,32 @@
+name: Docker build & push image
+
+on:
+  push:
+    branches: [v4]
+  workflow_dispatch:
+
+jobs:
+  docker:
```

Comment:
Can you make this workflow to only run if the repository is `jackyzha0/quartz`?

such that fork won't run this workflow.

Reply:
Ohh good point, I didn't think of that! However, GitHub actions are actually disabled by default on forked repositories, users need to enable them theirselves :)

![image](https://github.com/user-attachments/assets/e0e77d80-586d-48d5-a136-3d7c2f277431)

Reply:
So I think it would actually be worth leaving in, so that users who do want to host an image of their fork can do so, but it doesn't happen on accident. If you disagree I'll gladly change it though!

Reply:
I think even if the users enable workflow, this shouldn't be run by default.

if they want to then they can change it themselves. we don't want any surprise here.

Reply:
That's fair! I've updated the workflow to only run on `jackyzha0/quartz` :)